packages/loader/container-loader/src/deltaQueue.ts

```diff
@@ -56,7 +56,16 @@ export class DeltaQueue<T>
 		return this.processingPromise === undefined && this.q.length === 0;
 	}
 
-	public async waitTillProcessingDone() {
+	public async waitTillProcessingDone(): Promise<
+		| {
+				count: number;
+				duration: number;
+		  }
+		| {
+				count: number;
+				duration: number;
+		  }
+	> {
```

Comment:
I... what? 😆 Auto-fix doing weird things?
```suggestion
	public async waitTillProcessingDone(): Promise<
		{
			count: number;
			duration: number;
		}
	> {
```

Reply:
Wow, yup that's a weird one. Nice catch! Will update.